hw/ip/otbn/doc/_index.md

```diff
@@ -426,11 +426,73 @@ Rough expected process:
 
 ## Error conditions
 
-<div class="bd-callout bd-callout-warning">
-  <h5>Note</h5>
-
-  To be filled in as we create the implementation.
-</div>
+<table>
```

Comment:
Would it be possible to add the error identifiers (enum constants), such as ErrCodeBadDataAddr, back, to give us a "canonical identifier" that we can use in RTL and in the DIF.

Reply:
How do you feel about using the 'Name' column for the identifiers? It'd be nice to keep the column count down and separate names and identifiers does look a little redundant.

Reply:
We can do that, but in that case let's make the "Name" column an identifier (remove the spaces) so that there's no room for interpretation how to construct the identifier, and that we can easily grep for it.

Reply:
Yup that was the plan, ditch 'Name', have 'Identifier' instead.

I could do the otbn PKG and dif header updates as part of this PR so things are kept consistent everywhere?